tidy3d/components/mode.py

```diff
@@ -127,6 +127,15 @@ class ModeSpec(Tidy3dBaseModel):
         "yz plane, the ``bend_axis`` is always 1 (the global z axis).",
     )
 
+    bend_correction: bool = pd.Field(
+        False,
+        title="Bend correction",
+        description="Applies a correction by rotating the mode solution from a reference plane to"
+        "the actual mode plane. This is done by first computing the mode solution at a reference"
+        "plane normal to the bend's azimuthal direction, then rotating the field solution to"
+        "align with the mode plane, using the n_eff calculated at the reference plane.",
+    )
```

Comment:
I would try to make both the name of this field and the description a bit clearer. For example, it is not clear that this would only apply to the cases when both a bend and an angle is specificed.

Maybe something like:

```
    bend_angle_rotation: bool = pd.Field(
        False,
        title="Use rotation when both bend and angle are defined",
        description="Defines how modes are computed when both a bend and an angle are defined. If `False`, the two coordinate transformations are directly composed. If `True`, the structures in the simulation are first rotated, to compute a mode solution at a reference plane normal to the bend's azimuthal direction. Then, the fields are rotated to align with the mode plane, using the `n_eff` calculated at the reference plane. The second option can produce more accurate results, but more care must be taken for example in ensuring that the original mode plane intersects the right geometries in the simulation with rotated structures.",
    )
```